src/connectors/README.md

```diff
@@ -13,18 +13,22 @@ Connects provided component with a wrapper behind internal PWA routing actions.
 Everywhere as a React component rendered within the PWA app.
 
 #### Props provided
-- `historyPush(pathname: string)` - Opens new page in a safe way. Detects link types. Internal links (deep links) which are handled by the PWA app. External links are handled by opening the in-app-browser.
-- `historyReplace(pathname:string)` - Replaces current page with provided pathname if pathname is an internal link. If pathname is external link it works exactly as `historyPush`. If you intend to replace your current page with in-app-browser, use combination of `historyPush` and `historyPop` instead.
+- `historyPush(pathname: string, options: object[optional])` - Opens new page in a safe way. Detects link types. Internal links (deep links) which are handled by the PWA app. External links are handled by opening the in-app-browser.
+    - `state (object[optional])` - Property for options. Route state passed from history action. Is NOT the redux state. It is metadata only for the routing transition.
```

Comment:
It's confusing. `state` is an option property or it's a mistake and should be options?

Reply:
State is a property of options. I think it is clearer with updated commit